src/Temporalio/Workflows/ActivityOptions.cs

```diff
@@ -72,6 +72,13 @@ public class ActivityOptions : ICloneable
         /// </summary>
         public string? TaskQueue { get; set; }
 
+        /// <summary>
+        /// Gets or sets a single-line fixed summary for this activity that may appear in UI/CLI.
+        /// This can be in single-line Temporal markdown format.
+        /// </summary>
+        /// <remarks>WARNING: This setting is experimental.</remarks>
+        public string? Summary { get; set; }
```

Comment:
Is this one not static?

Reply:
No, and neither is the timer one. And technically in history, neither is the workflow one. We only call the workflow one `StaticSummary` because we want to use the term `StaticDetails` and to clarify they are immutable/static and differentiate those from the concept of `CurrentDetails` which is mutable. So workflows have "static" summary/details even though they are really just details, but we don't want confused with "current" details. We could not find better terminology for these when we designed them.